tools/patch_skia.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# Copyright (c) 2015 Intel Corp. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""
+patch_skia.py -- Apply patches to skia when running gclient.
+"""
+
+import os
+import subprocess
+import sys
+
+
+def main(argv=None):
+  if argv is None:
+    argv = sys.argv
+
+  tools_dir = os.path.dirname(os.path.abspath(__file__))
+  xwalk_dir = os.path.dirname(tools_dir)
+  # tools_dir should be at src/xwalk/tools/patch_skia.py
+  # so src is at tools_dir/../../../
+  src_dir = os.path.dirname(xwalk_dir)
+  root_dir = os.path.dirname(src_dir)
+  third_party_dir = os.path.join(src_dir, 'third_party')
+  skia_dir = os.path.join(third_party_dir, 'skia')
+  patch_dir = os.path.join(root_dir, argv[1])
+
+  output = subprocess.check_call(["git", "reset", "--hard"], cwd=skia_dir)
```

Comment:
I think you're looking for `subprocess.call()`. If `check_call()` fails it will raise an exception.